scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/input_model/data/control_plane.yml

```diff
@@ -65,7 +65,7 @@
           allocation-policy: any
 {%     endif %}
           service-components:
-{%     for service_component in ns.service_components|unique|sort %}
+{%     for service_component in ns.service_components|unique|sort if (versioned_features[service_component.split('-')[0]]|default(dict()))['enabled']|default(true) %}
```

Comment:
This only takes care of the generated input models. We need the same thing for the case using `ardana-input-models` input models, meaning you need to parse and modify an existing input model.